tests/react/use-fireproof.test.tsx

```diff
@@ -73,93 +73,6 @@ describe("HOOK: useFireproof useLiveQuery has results", () => {
   });
 });
 
-// Separate describe block for array-like behavior test
-describe("HOOK: useFireproof useLiveQuery supports array-like behavior", () => {
```

Comment:
the ugly thing is the implementation for supporting misuse of the API. @mabels you convinced me we shouldnt do that but users might tell use we have to... in which case we can bring this stuff back